app/src/main/java/com/nervousfish/nervousfish/activities/VisualVerificationActivity.java

```diff
@@ -28,9 +28,11 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger("VisualVerificationActivity");
     private static final int SECURITY_CODE_LENGTH = 5;
+    private static final int NUM_BUTTONS = 12;
```

Comment:
> Everybody knows how many tiles there are on the screen so they can figure out that the picture's animals and objects aren't randomly placed and can more easily seatup for a brute force attack.

You mean that they can now find out how many tiles there are on the screen? They already could, the code for that is also part of this repository.

Also, the intention is to have multiple different images for the visual verification method, making already a bit more difficult to do a brute force attack